pydantic_factories/constraints/date.py

```diff
@@ -0,0 +1,31 @@
+from datetime import date, timedelta
+from typing import TYPE_CHECKING, Optional, cast
+
+if TYPE_CHECKING:
+    from faker import Faker
+    from pydantic import ConstrainedDate
+
+
+def handle_constrained_date(constrained_date: "ConstrainedDate", faker: "Faker") -> date:
+    """
+    Generates a date value fulfilling the expected constraints.
+    Args:
+        constrained_date:
+        faker:
+
+    Returns:
+
+    """
+    start_date: Optional[date] = date.today() - timedelta(days=100)
```

Comment:
I might be wrong but it looks like `start_date` and `stop_date` can not be `None` - hence, their type annotations can be just `date`.

But I might be wrong as I am looking at the code via Github web interface.

Reply:
fixed